kramer/src/main/kotlin/kramer/Templates.kt

```diff
@@ -58,7 +72,7 @@ internal fun mavenAarTemplate(
 # $coordinate raw classes
 raw_jvm_import(
     name = "${target}_classes",
-    jar = "$fetchRepo:classes.jar",$jetify
+    jar = "$fetchRepo",$jetify
```

Comment:
We don't need classes.jar?

Reply:
No - we are passing in the filegroup target (e.g. @androidx_core_core//maven). That filegroup includes the classes.jar and the -sources.jar, and the raw_jvm_import rule knows how to get the right bits out.